src/templates/gwfcore/gwfcore-batch.template.yaml

```diff
@@ -135,7 +162,7 @@ Resources:
         # This is the maximum price for spot instances that Batch will launch.
         # Lowering this puts a limit on the spot capacity that Batch has available.
         # Spot instances are terminated when on-demand capacity is needed, regardless of the price set.
-        BidPercentage: 100 
+        BidPercentage: !Ref BatchSpotBidPercentage 
```

Comment:
It's considered best practice to set the Spot bid percentage to 100. Anything lower can artificially cap the spot capacity you allow the compute environment to access. You still get the available SPOT instance pricing for instances launched.

Reply:
@wleepang : We added this just to have an option to update it from parameters as there might be some cases where customers won't want to proceed if the spot prices are above certain level. The default anyways is 100.

Reply:
Ok. Not a blocker to getting this merged. We can address the caveats of setting the spot bid percentage below 100 in documentation.